keyboard-listener/src/main/java/com/rw/keyboardlistener/KeyboardUtils.java

```diff
@@ -21,7 +21,7 @@
 
     private SoftKeyboardToggleListener mCallback;
     private View mRootView;
-    private Boolean prevValue = null;
+    public static Boolean sPrevValue = null;
```

Comment:
Hi, why does this have to be public and static? This should not be accessed from outside.